lib/preview/link.js

```diff
@@ -36,6 +36,11 @@ var PreviewConstants = require('./constants');
 var PreviewImage = require('./image');
 var Result = require('./model').Result;
 
+// URLs matching any of these regexes will be marked "not embeddable" over all protocols.
+var DISALLOW_EMBED_REGEXES = [
+  /^http(s)?:\/\/(www\.)?portfolium\.com\/entry/
```

Comment:
@paulkerschen the [Portfolium API docs](https://api.portfolium.com/) have sample URLs starting with `https://portfolium.cloudimg.io`.  Do we need to account for this or other patterns?